### PR TITLE
docs: add missing serialization yaml documentation

### PR DESCRIPTION
**Pull request by johnkrovitch**
Add missing serialization yaml documentation

### CODE REVIEW

File: core/serialization.md [hunk 1]
```diff
@@ -196,6 +198,33 @@ class Book
...
+    properties:
+        name:
+            groups: ['get', 'put']
+
+# config/serializer/Book.yaml
+App\Entity\Book:
+    attributes:
+        name:
+            groups: ['get', 'put']
+
```
Comment (alanpoulain) [inline core/serialization.md:221]: Line to remove?

File: core/serialization.md [hunk 2]
```diff
@@ -196,6 +198,33 @@ class Book
...
+App\Entity\Book:
+    attributes:
+        normalization_context:
+            groups: ['get']
+    itemOperations:
+        get: ~
+        put:
+            normalization_context:
+                groups: ['put']
+    properties:
```
Comment (alanpoulain) [inline core/serialization.md:212]: I'm not sure to understand this. Shouldn't it be in the serializer configuration?

File: core/serialization.md [hunk 3]
```diff
@@ -408,10 +496,33 @@ class Person
...
+            groups: ['person']
+
+# config/serializer/Person.yaml
+App\Entity\Person:
+    attributes:
+        name:
+            groups: ['person']
+        parent:
+            readableLink: false
+            writableLink: false
```
Comment (alanpoulain) [inline core/serialization.md:515]: No this is not right. It's an API Platform configuration.

File: core/serialization.md [hunk 4]
```diff
@@ -470,12 +581,31 @@ class Greeting
...
+App\Entity\Greeting:
+    collectionOperations:
+        get:
+            normalization_context:
+                groups: 'greeting:collection:get'
+
+# config/serializer/Greeting.yaml
+App\Entity\Greeting:
+    attributes:
+        getSum:
```
Comment (alanpoulain) [inline core/serialization.md:595]: `id` is missing

File: core/serialization.md (hunk 2 above)
Comment (johnkrovitch) [reply core/serialization.md:212]: This is used for example to have different serializations groups by API Platform operation. The serializer configuration is used to configure which field will be displayed according to the group defined in the operation.

File: core/serialization.md [hunk 5]
```diff
@@ -470,12 +580,33 @@ class Greeting
...
+    collectionOperations:
+        get:
+            normalization_context:
+                groups: 'greeting:collection:get'
+
+# config/serializer/Greeting.yaml
+App\Entity\Greeting:
+    attributes:
+        id:
+            groups: 'greeting:collection:get'
```
Comment (alanpoulain) [inline core/serialization.md:595]: `name` is missing as well.

Comment (johnkrovitch) [reply core/serialization.md:595]: added

**Comment by alanpoulain** [comment]
Thanks @johnkrovitch.

**Pull request merged**